src/zzz_od/application/hollow_zero/lost_void/lost_void_app.py

```
@@ -202,7 +203,20 @@ def check_bounty_commission_before(self) -> OperationRoundResult:
     @node_from(from_name='识别悬赏委托完成进度', status=STATUS_AGAIN_MATRIX)
     @operation_node(name='矩阵行动-前往入口')
     def matrix_goto_entry(self) -> OperationRoundResult:
-        return self.round_by_goto_screen(screen_name='迷失之地-矩阵行动-编队选择')
+        # 正常流程：前往编队选择。点"下一步"后可能弹出确认弹窗（中途加入续传场景），
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)